python-client/giskard/push/perturbation.py

```diff
@@ -167,9 +168,16 @@ def _text(
 ):
     passed = False
     # Iterate over the possible text transformations
-    for text_transformation in text_transfo_list:
+    for text_transformation in text_transformation_list:
         # Create the transformation
-        t = text_transformation(column=feature)
+        _is_typo_transformation = issubclass(text_transformation, TextTypoTransformation)
+        kwargs = {}
+        if _is_typo_transformation:
+            hashed_seed = hash(f"{', '.join(map(lambda x: repr(x), ds_slice_copy.df.values))}".encode("utf-8"))
+            positive_hashed_seed = hashed_seed % ((sys.maxsize + 1) * 2)
```

Comment:
@rabah-khalek could you add a commend why we're doing it? In a week we won't remember by heart